common/ethereum/transaction.py

```diff
@@ -12,7 +12,7 @@
 from ..utils.format import hex_to_bytes
 
```

Comment:
Please run `black` utility to format the source code.

Reply:
done